pkgs/by-name/yu/yunfaavatar/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  pciutils,
+  x11Support ? true,
+  fetchpatch,
+  bash,
+  imagemagick,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "yunfaavatar";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yunfachi";
+    repo = "yunfaAvatar";
+    rev = version;
+    hash = "sha256-JbolO1I4K9+RBFJ/U611En4Rx5cG2+z1IcxYEeiaq6Q=";
+  };
+
+  outputs = ["out" "man"];
+
+  strictDeps = true;
+  buildInputs = [bash imagemagick];
+  nativeBuildInputs = [makeWrapper];
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "SYSCONFDIR=${placeholder "out"}/etc"
+  ];
+
+  meta = with lib; {
+    description = "Utility for automatic centralized changing of avatar in Github, Discord, Steam, Hypixel, and many more";
+    homepage = "https://github.com/yunfachi/yunfaAvatar";
+    license = licenses.unfree;
+    platforms = platforms.all;
+    maintainers = with maintainers; [yunfachi];
+    mainProgram = "yunfaavatar";
```

Comment:
This doesn't need to be set if it matches the package name.

Reply:
that's not true.

Reply:
My bad.